apps/src/applab/applab.js

```diff
@@ -586,6 +592,11 @@ Applab.init = function(config) {
         e.stop();
       });
     }
+    
+    if (studioApp.share) {
+      // automatically run in share mode:
+      window.setTimeout(studioApp.runButtonClick, 0);
```

Comment:
oops, this is applab-specific... may be a good idea anyway so that we can reference "this" in Applab.prototype.runButtonClick in the future.